src/View/User/Includes/_course_tab.html.twig

```diff
@@ -0,0 +1,82 @@
+{% block course_tab %}
+<div class="course-tab-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+    <div class="tab_title_text">
+        <p class="tab_title">
+            CONDITIONS
+        </p>
+        <p class="tab_text">
+            </br> Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
```

Comment:
same has before and repeat this for the of your user story

Reply:
Done